src/main/java/sirius/biz/storage/layer3/VirtualFileSystemController.java

```diff
@@ -222,6 +223,41 @@ public void delete(WebContext webContext) {
         }
     }
 
+    /**
+     * Deletes the given list of files or directories.
+     *
+     * @param webContext the request to handle
+     */
+    @LoginRequired
+    @Routed("/fs/delete/multiple")
+    @Permission(PERMISSION_VIEW_FILES)
+    public void deleteMulitple(WebContext webContext) {
+        List<String> filePaths = Arrays.stream(webContext.get("paths").asString().split("[,;]"))
```

Comment:
Do we really need to split manually? With a regex that is not compiled? Could that be done via `WebContext.getParameters()`?

Reply:
As Jakob mentioned we also have the possibility to send a parameter multiple times and get all values via `getParameters`. This is the preferred way.